src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -105,4 +111,13 @@ object TestUtils  {
     ps.setTimestamp(4, Timestamp.from(FixedTimeSource.now))
     ps.executeUpdate()
   }
+
+  def addAntivirusMetadata(fileId: String, consignmentId: String): Unit = {
+    val sql = s"insert into AVMetadata (FileId, Result, Datetime) VALUES (?, ?, ?)"
+    val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
+    ps.setString(1, s"${fileId}")
+    ps.setString(2, consignmentId)
```

Comment:
It looks like the second parameter of this INSERT statement should be the antivirus result, rather than the consignment ID.

Reply:
Ah you're right! Will change :ok_hand: